frontend/src/constants/environments.ts

```diff
@@ -1,36 +1,43 @@
+import { stringToBoolean } from "src/utils/generalUtils";
+
 const {
   NODE_ENV,
   NEXT_PUBLIC_BASE_PATH,
-  USE_SEARCH_MOCK_DATA = "",
+  USE_SEARCH_MOCK_DATA,
   SENDY_API_URL,
   SENDY_API_KEY,
   SENDY_LIST_ID,
   API_URL,
-  API_AUTH_TOKEN = "",
+  API_AUTH_TOKEN,
+  NEXT_BUILD,
+  SESSION_SECRET,
   NEXT_PUBLIC_BASE_URL,
-  FEATURE_SEARCH_OFF = "false",
-  FEATURE_OPPORTUNITY_OFF = "false",
-  NEXT_BUILD = "false",
-  SESSION_SECRET = "",
+  FEATURE_SEARCH_OFF,
+  FEATURE_OPPORTUNITY_OFF,
+  FEATURE_AUTH_OFF,
 } = process.env;
 
+export const featureFlags = {
```

Comment:
this could be done elsewhere if we'd rather keep this file clean

Reply:
They touch the envars so seems good to me.